Assignment-1/models/bill.go

```diff
@@ -0,0 +1,50 @@
+package models
+
+type Bill struct {
+	ItemName     string
+	ItemPrice    float64
+	SalesTax     float64
+	FinalPrice   float64
+	ItemQuantity int
+}
+
+func (b *Bill) Getter() Bill {
```

Comment:
why this is needed?
Bill already is publicly accessible.

Reply:
fixed it